### PR TITLE
[FLINK-24818][API/State Processor]Correct the ttlConfig comment in StateDescriptor

### PR DESCRIPTION
**Pull request by datayjz**
## What is the purpose of the change

This pull request fixes the comment error of the ```ttlConfig``` field in ```StateDescriptor```.

## Brief change log
  - *Correct the ```ttlConfig``` comment in ```StateDescriptor```.*

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (no)
  - The S3 file system connector: (no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: flink-core/src/main/java/org/apache/flink/api/common/state/StateDescriptor.java [hunk 1]
```diff
@@ -100,7 +100,9 @@
     /** Name for queries against state created from this StateDescriptor. */
     @Nullable private String queryableStateName;
 
-    /** Name for queries against state created from this StateDescriptor. */
+    /**
+     * The state time-to-live(TTL). If the setting is not displayed, the ttl will not be turned no.
```
Comment (Myasuka) [inline flink-core/src/main/java/org/apache/flink/api/common/state/StateDescriptor.java:104]:
```suggestion
     * The configuration of state time-to-live(TTL), it is disabled by default.
```

**Review by Myasuka — CHANGES_REQUESTED** [review]
Thanks for the contribution, I think the suggestion is better.

File: flink-core/src/main/java/org/apache/flink/api/common/state/StateDescriptor.java (hunk 1 above)
Comment (datayjz) [reply flink-core/src/main/java/org/apache/flink/api/common/state/StateDescriptor.java]: Thank you for you recovery. I also think you suggestion is more concise.

**Comment by datayjz** [comment]
hi, @Myasuka could you review it for me. thanks

**Review by Myasuka — CHANGES_REQUESTED** [review]
Thanks for the update, the CI is still broken, please run `mvn spotless:apply -pl :flink-core` first before updating this PR.

**Comment by datayjz** [comment]
@flinkbot run azure re-run the last Azure build

**Comment by datayjz** [comment]
@Myasuka thanks.  The code has been reformatted, the CI is passed.

**Review by Myasuka — APPROVED** [review]
LGTM

**Comment by Myasuka** [comment]
@datayjz , if you plan to contribute for the next time, plase make the commit history clean and make the commit message foramt like `[FLINK-xxxx][state] xxxxxxx`.

**Comment by datayjz** [comment]
> @datayjz , if you plan to contribute for the next time, plase make the commit history clean and make the commit message foramt like `[FLINK-xxxx][state] xxxxxxx`.

Ok, I'll contribute according to the corresponding format next time. Thank you for your gidance.

**Pull request closed without merging**